src/ts/utils/slider.ts

```diff
@@ -0,0 +1,43 @@
+export const getNumberOfSteps = (step: number, min: number, max: number) => {
+  const totalRange = max - min;
+  return Math.floor(totalRange / step) + 1;
```

Comment:
Prevents negative step count and removes a lil code.

```suggestion
  const totalRange = Math.abs(max - min);
  return Math.ceil(totalRange / step);
```